src/controllers/handlers/statsHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { HandlerContextWithPath } from "../../types"
+import { S3 } from "aws-sdk";
+
+export async function statsHandler({ components: { config } }: Pick<HandlerContextWithPath<"config", "/stats">, "components" | "params" | "url">) {
+  const region = await config.requireString("AWS_REGION");
+  const s3 = new S3({ region })
```

Comment:
In later PRs, it maybe be better to get this s3 instance as a backup, for testing purposes